docs/src/reference.md

```diff
@@ -0,0 +1,121 @@
+# Reference
+
+The API reference of `NNlib`.
+
+## Activation Functions
+
+Non-linearities that go between layers of your model. Note that, unless otherwise stated, activation functions operate on scalars. To apply them to an array you can call `σ.(xs)`, `relu.(xs)` and so on.
```

Comment:
Is this true? I think a PR gave them all broadcasting behaviour:
```
julia> relu(-2:2)
5-element Vector{Int64}:
 0
 0
 0
 1
 2
```
This is slightly weird, and won't work for `tanh`. I think the logic was that this was less ugly than introducing a special `Activate` layer, or writing `x -> relu.(x)` in your Chain.

Reply:
Out of curiosity, why is the behavior implemented differently for `tanh`?

Reply:
Because some functions already have matrix input definitions that NNlib does not override.

Reply:
I don't have an issue with the current wording because it's not clear whether/how much we want to advertise this behaviour. We can always relax it later on.